ambry-messageformat/src/main/java/com/github/ambry/messageformat/MessageFormatRecord.java

```diff
@@ -1756,6 +1754,78 @@ public static BlobData deserializeBlobRecord(CrcInputStream crcStream) throws IO
     }
   }
 
+  /**
+   *  - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+   * |         |           |              |            |            |            |
+   * | version | blobType  | isCompressed |    size    |  content   |     Crc    |
+   * |(2 bytes)| (2 bytes) | (1 byte)     |  (8 bytes) |  (n bytes) |  (8 bytes) |
+   * |         |           |              |            |            |            |
+   *  - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+   *  version    - The version of the blob record
```

Comment:
why 1 byte for a boolean ?

Reply:
that's the pattern that is common used Ambry.  Use 1 byte for a boolean.